combination.ipynb

```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
```

Comment:
Indicate time point of these plots. Some explanation of them at the top would be helpful, too.

Reply:
Sure.